dispatcher/pool.py

```diff
@@ -186,7 +189,12 @@ async def manage_timeout(self) -> None:
             self.events.timeout_event.clear()
 
     async def up(self) -> None:
-        process = self.process_manager.create_process((self.next_worker_id,))
+        process = self.process_manager.create_process(
+            (
+                self.settings_stash,
```

Comment:
We will need this in order to enable use of forkserver for creating worker processes. Because we can't rely on references to global objects from the parent in that model. So we have to pass data that we need.